.circleci/dcm4chee/upload_example_data.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import urllib.request
+from io import BytesIO
+
+from dicomweb_client import DICOMwebClient
+from pydicom import dcmread
+
+
+def upload_example_data(server_url):
+
+    # This is TCGA-AA-3697
+    download_urls = [
+        'https://data.kitware.com/api/v1/file/6594790e9c30d6f4e17c9d0d/download',
```

Comment:
I'd prefer to use the hashsum download rather than the file id download, that way if the files are moved they can still be accessed (that is, use `https://data.kitware.com/api/v1/file/hashsum/{algo}/{hashvalue}/download`).

Reply:
Done!